movie.sh

```diff
@@ -1,55 +1,58 @@
 #!bin/bash
 
-function getrating()
+function add_style()
 {
-#Fetch tuple from the Omdb API
-curl -s  "http://www.omdbapi.com/?t=$1&y=&plot=short&r=xml" > mov.txt
-#Getting the Line with the information
-grep -o imdbRating=".*" mov.txt > submov.txt
-input="submov.txt"
-echo ""
-ctr=0
-rating=""
-#Extracting the rating. Since the format is XYZ= "x"
-#Track the opening and closing quotes to get x which is rating
-#ctr tracks number of " encountered
-while IFS= read -r -n1 var
-do
-  if [ $ctr -eq 2 ]
-  then
-        break
-  elif [ $var == '"' ]
-  then
-        ctr=$(( $ctr + 1 ))
-  elif [ $ctr -eq 1 ]
-  then
-        rating="$rating$var"
-  else
-        continue
-  fi
-done < "$input"
-rm mov.txt
-rm submov.txt
+        tput bold
+        tput setaf $2
+        echo $1
+        tput sgr 0
+        return 0
+}
+
+mkdir tmp
+function get_rating()
+{
+response=$( curl -s  "http://www.omdbapi.com/?t=$1&y=&plot=short&r=xml" | grep -o imdbRating=".*" | cut -c 13,14,15)
 #Convert String to Float
```

Comment:
All comments should be on the same level of indentation as the code it refers to. All comments must start with `# ` (i.e. a hash symbol and a space).